src-built-in/components/apitest/test1.js

```diff
@@ -0,0 +1,1538 @@
+		triggerGetWindowTitle();
+	});
+	$("#Windows").append(getWindowTitle);
+	var estHeaderCommandChannel = $("<estHeaderCommandChannel>Establish Header Command Channel</estHeaderCommandChannel>");
+	estHeaderCommandChannel.click(function () {
+		triggerEstHeaderCommandChannel();
+	});
+	$("#Windows").append(estHeaderCommandChannel);
+	var injectHeader = $("<injectHeader>Inject Header</injectHeader>");
+	injectHeader.click(function () {
+		triggerInjectHeader();
+	});
+	$("#Windows").append(injectHeader);
+	var maximize = $("<maximize>Maximize</maximize>");
+	maximize.click(function () {
+		triggerMaximize();
+	});
+	$("#Windows").append(maximize);
+	var minimize = $("<minimize>Minimize</minimize>");
+	minimize.click(function () {
+		triggerMinimize();
+	});
+	$("#Windows").append(minimize);
+	var restore = $("<restore>Restore</restore>");
+	restore.click(function () {
+		triggerRestore();
+	});
+	$("#Windows").append(restore);
+	var sendWinIdentifierForTilingOrTabbing = $("<sendWinIdentifierForTilingOrTabbing>Send Window Identifier For Tiling or Tabbing</sendWinIdentifierForTilingOrTabbing>");
+	sendWinIdentifierForTilingOrTabbing.click(function () {
+		triggerSendWinIdentifierForTilingOrTabbing();
+	});
+	$("#Windows").append(sendWinIdentifierForTilingOrTabbing);
+	var setAlwaysOnTop = $("<setAlwaysOnTop>Set Always On Top</setAlwaysOnTop>");
+	setAlwaysOnTop.click(function () {
+		triggerSetAlwaysOnTop();
+	});
+	$("#Windows").append(setAlwaysOnTop);
+	var setWindowTitle = $("<setWindowTitle>Set Window Title</setWindowTitle>");
+	setWindowTitle.click(function () {
+		triggerSetWindowTitle();
+	});
+	$("#Windows").append(setWindowTitle);
+	var showAtMousePos = $("<showAtMousePos>Show At Mouse Position</setWindowTshowAtMousePositle>");
+	showAtMousePos.click(function () {
+		triggerShowAtMousePos();
+	});
+	$("#Windows").append(showAtMousePos);
+	var startTilingOrTabbing = $("<startTilingOrTabbing>Start Tiling Or Tabbing</startTilingOrTabbing>");
+	startTilingOrTabbing.click(function () {
+		triggerStartTilingOrTabbing();
+	});
+	$("#Windows").append(startTilingOrTabbing);
+	var stopTilingOrTabbing = $("<stopTilingOrTabbing>Stop Tiling Or Tabbing</stopTilingOrTabbing>");
+	stopTilingOrTabbing.click(function () {
+		triggerStopTilingOrTabbing();
+	});
+	$("#Windows").append(stopTilingOrTabbing);
+
+	/* Dialog Buttons */
+	var openDialog = $("<openDialog>Open Dialog</openDialog>");
+	openDialog.click(function () {
+		triggerOpenDialog();
+	});
+	$("#Dialog").append(openDialog);
+
+	/* Search Client Buttons */
+	var search = $("<search>search</search>");
+	search.click(function () {
+		triggerSearch();
+	});
+	$("#Search").append(search);
+}
+/* SeachClient Functions*/
+function triggerSearch() {
+	if (document.getElementById('symbolInput').value != '')
+		searchClient.search({
+			text: document.getElementById('symbolInput').value
+		}, function (err, response) {
+			if (!err) {
+				alert('Sussceed. See console for detail.')
```

Comment:
Could you add test next to the button with message rather than using alerts?
```suggestion
				alert('Success. See console for detail.')
```